cmd/cluster-agent-cloudfoundry/app/app.go

```diff
@@ -201,6 +201,14 @@ func run(cmd *cobra.Command, args []string) error {
 		return log.Errorf("Error while starting agent API, exiting: %v", err)
 	}
 
+	if config.Datadog.GetBool("cluster_agent.admission_controller.enabled") {
+		log.Info("Starting admission controller server")
+
+		if err = admissioncontroller.StartServer(); err != nil {
+			return log.Errorf("Error while starting admission controller server, exiting: %v", err)
+		}
+	}
```

Comment:
I'd like some verification if `cluster-agent-cloudfoundry` should be modified here.
It seems like it follows the same implementation was `cluster-agent`, so that made me assume that we'd like to introduce the webhook here too, but I'm not sure of the implications.